tests/integration/test_builder.py

```diff
@@ -28,7 +28,10 @@ def test_exporter_builder():
 save_path: {}
 
 pipeline:
-  dataset: !SSTDataset
+  dataset: !TabularDataset.from_path
+    train_path: tests/data/dummy_tabular/train.csv
```

Comment:
This paths are not working also... I wonder where the ray processes are being triggered.

Reply:
Solved in #153